.github/workflows/build-check-install.yaml

```diff
@@ -50,6 +50,24 @@ on:
         required: false
         default: false
         type: boolean
+      check-note-blocklist:
+        description: Whether to check for NOTEs that should cause the pipeline to fail
+        required: false
+        default: false
+        type: boolean
+      note-blocklist:
+        description: List of regular expressions appearing in NOTEs that should cause the pipeline to fail
+        # Example usage:
+        # note-blocklist: >-
+        #   c("checking dependencies in R code ... NOTE( )+Namespace in Imports field not imported from",
+        #   "checking package dependencies ... NOTE( )+Depends: includes the non-default packages",
+        #   "checking R code for possible problems ... NOTE( )+.*: no visible global function definition for",
+        #   "checking for unstated dependencies in vignettes ... NOTE( )+\'.*\' import not declared from",
+        #   "checking installed package size ... NOTE( )+installed size is")
+        required: false
+        default: c("")
+        type: string
```

Comment:
I would actually let this be a multiline string
And then the multiline string gets passed to R where it gets converted to a vector.
We should not mix data types like this.

Reply:
Fixed this.